<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticeDoctorSettings</class>
 <widget class="QWidget" name="PracticeDoctorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PracticeDoctorSettings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="doctorList"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addDoctor">
       <property name="text">
        <string>Добави доктор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeDoctor">
       <property name="text">
        <string>Премахни доктор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="adminCheck">
       <property name="text">
        <string>Администратор</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
